### PR TITLE
Ypersky clone with multiple files

### PR DESCRIPTION
**Pull request by ypersky1980**
The current PR adds a test : creation of number of samples of Clones for a PVC with multiple files ( ~1 million files). 
The test creates clones for such PVC, calculates creation and deletion times of the samples , also CSI creation and deletion times, and creation /deletion/csi creation/csi deletion averages.
The test sends the results to the ES server and uploads those results to the Performance Dashboard.

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py [hunk 1]
```diff
@@ -336,3 +325,302 @@ def init_full_results(self, full_results):
...
+        Test assign nodeName to a pod using RWX pvc
+        Each kernel (unzipped) is 892M and 61694 files
+        The test creates a pvc and a pods, writes kernel files multiplied by number of copies
+        The test creates number of clones samples, calculates creation and deletion times for each one the clones
+        and calculates the average creation and average deletion times
+        """
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.5.tar.gz"
+        download_path = "tmp"
+
+        test_start_time = PASTest.get_time()
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:379]: use `self.get_time()`

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py [hunk 2]
```diff
@@ -336,3 +325,302 @@ def init_full_results(self, full_results):
...
+                self.uuid,
+                self.crd_data,
+                self.full_log_path,
+                "test_pvc_clone_performance_multiple_files_fullres",
+            )
+        )
+
+        full_results.add_key("files_number", files_written)
+
+        test_end_time = PASTest.get_time()
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:566]: same as above, use `self.get_time()`

**Review by Avilir — APPROVED** [review]
LGTM

**Comment by keemano** [comment]
LGTM

**Review by ramkiperiy — APPROVED** [review]

**Pull request merged**